docs/05-technical-reference/00-configuration-parameters/obsv-01-configpara-observability.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Observability charts
+---
+
+You find the configurable parameters in charts and subcharts in the respective `values.yaml` files. The `values.yaml` files are fully documented and provide details on the configurable parameters and their customization.
+
+To override the configuration, redefine the default values in your custom `values.yaml` file and [deploy them](../../04-operation-guides/operations/03-change-kyma-config-values.md).
```

Comment:
I'm not pushing 'cause you're describing multiple charts here, but all the other chart and sub-chart documents contain this exact phrase, so I would probably be a good idea to be consistent here. 
![image](https://user-images.githubusercontent.com/53601578/133340674-bdd5c067-eff0-4965-88a4-fea03754304c.png)
```suggestion
To configure the Observability charts, override the default values of the respective `values.yaml` files. This document links to the charts with the parameters that you can configure.

> **TIP:** See how to [change Kyma settings](../../04-operation-guides/operations/03-change-kyma-config-values.md).
```

Reply:
This sentenced was rephrased during the review. We want to point out that users must not touch the original values.yaml but their custom one. I'll think about a way to phrase this that is more consistent to our template but keeps that distinction.

Reply:
Actually, I propose to switch the default sentence. The current default text isn't as concise as it could be.